library/objective-c/BUILD

```diff
@@ -8,12 +8,13 @@ exports_files([
 objc_library(
     name = "envoy_engine_objc_lib",
     srcs = [
-        "EnvoyEngine.mm",
+        "EnvoyEngine.m",
+        "EnvoyHttpStream.m",
     ],
     hdrs = [
         "EnvoyEngine.h",
+        "EnvoyHttpStream.h",
```

Comment:
I think this should be moved to `exports_files` (maybe this needs to be rebased?)